include/kos/mutex.h

```diff
@@ -167,6 +167,29 @@ int mutex_destroy(mutex_t *m);
 */
 int mutex_lock(mutex_t *m);
 
+/** \brief  Lock a mutex.
+
+    This function will lock a mutex, if it is not already locked by another
+    thread. If it is locked by another thread already, this function will block
+    until the mutex has been acquired for the calling thread.
+    This function can be from within an interrupt context. In that case, if the
```

Comment:
```suggestion
    This function can be from within an interrupt context. In that case, if the
```
This function can be called?